tools/src/buf.txt

```diff
@@ -0,0 +1 @@
+github.com/bufbuild/buf/cmd/buf@v1.28.1
```

Comment:
maybe we can remove these file, define thing directly in makefile, wdyt @arkodg ?